examples/resumption/ColdResumption_Client.cpp

```diff
@@ -25,10 +25,10 @@ namespace {
 class HelloSubscriber : public virtual Refcounted, public Subscriber<Payload> {
  public:
   void request(int n) {
-    while (!subscription_) {
+    while (!Subscriber<Payload>::subscription()) {
```

Comment:
This can be `folly::Baton<>::wait()`, and `onSubscribe()` can do `folly::Baton<>::post()`.